### DISCUSSION TITLE
Poseidon: Feature Request: Horizontal menu

### DISCUSSION THREAD

**Discussion by chsreedhar** [primefaces/community#2132] [PrimeFaces Templates]
Am planning to purchase Poseidon Layout.  But found horizontal menu of Freya Layout is most suitable to my requirement.
Is it possible to make Freya Layout Horizontal menu in Poseidon Layout,. i.e Topbar and Menu together.

**Comment by tandraschko** [comment]
thats not possible without adjusting the sources by yourself, sorry
